.github/actions/spelling/expect.txt

```diff
@@ -0,0 +1,280 @@
+AClass
+Adafruit
+admins
+ALLEXTERNALS
+amsmath
+ANamespace
+arduino
+ASingle
+ATTRS
+AUTOBRIEF
+AUTOGEN
+AUTOLINK
+autotune
+avr
+batchmode
+bibtex
+Bierwagen
+biodiversity
+bitmaps
+BUGLIST
+callergraph
+callgraph
+cassert
+Celcius
+CFG
+cgi
+charset
+ci
+Clib
+CMake
+CODEFILE
+COLORSTYLE
+colorwheel
+config
+Controlb
+cout
+cpp
+css
+CTORS
+cxx
+datetime
+ddl
+defualt
+DEPRECATEDLIST
+DHTML
+diafile
+DIRS
+Dnominal
+docbook
+docset
+DOCTYPE
+dontinclude
+dotfile
+DOTFONTPATH
+dox
+doxyfile
+doxygen
+doxygenversion
+doxyindexer
+doxyrules
+doxysearch
+Doxywizard
+DVI
+ECLIPSEHELP
+eep
+endcond
+endif
+endl
+enum
+exe
+FEEDNAME
+Firefox
+FONTNAME
+FONTPATH
+FONTSIZE
+forkme
+func
+gcc
+gdiplus
+Gemfile
+github
+gmail
+godmode
+google
+Graphviz
+grayscales
+gtags
+gz
+hh
+hhc
+hhk
+hhp
+hidecallergraph
+hidecallgraph
+hideinitializer
+HIGHTHRESH
+hlp
+homepage
+hpp
+href
+htags
+htm
+html
+HTMLHELP
+http
+hundreth
+hxx
+hyperlink
+idl
+ieeetr
+ifdef
+img
+IMPLIMENT
+ine
+inl
+ino
+intlimits
+iostream
+ipp
+itcl
+itoa
+ixx
+javadoc
+javascript
+Jax
+jgfoster
+jpg
+Kd
+Kp
+labled
+libc
+libclang
+libiconv
+LOGFILE
+LOWTHRESH
+mailto
+MAINPAGE
+makeindex
+MAKEVAR
+mathjax
+MDFILE
+mega
+MEMB
+memset
+microcontroler
+microsoft
+Microsoft
+millis
+MML
+MOSI
+Mozilla's
+msc
+mscfile
+mscgen
+MULTILINE
+mycompany
+myfile
+myproject
+namespace
+NONINFRINGEMENT
+NSPACES
+nullptr
+numwires
+odl
+odo
+OMG's
+onthank
+Onthank
+openacidificationproject
+osubgrouping
+OSX
+OVUV
+pageview
+PARAMDOC
+pcb
+pdf
+pdflatex
+perl
+PERLMOD
+php
+phtml
+plainnat
+plantuml
+plugin
+plugins
+png
+popen
+porportional
+powersupply
+pragma
+PREDEF
+prestoncarman
+println
+PROGRAMLISTING
+projectbrief
+projectlogo
+projectname
+projectnumber
+propget
+propput
+px
+py
+pyw
+qch
+qhelpgenerator
+QHG
+qhp
+qsf
+rb
+README
+REFIN
+REFINHIGH
+REFINLOW
+relpath
+requirments
+Resistore
+resitance
+rpc
+rtd
+RTDIN
+RTDINLOW
+rtf
+searchdata
+SEARCHENGINE
+seo
+setpoint
+showinitializer
+sideeffect
+sizeof
+Smath
+src
+Ssymbols
```

Comment:
I can't find this in the repository. Is it part of `AMSsymbols`?

Reply:
All the "words" listed in this file were suggested by the spelling action. It seems that maybe installed libraries or other sources outside our code are included. Although I am not sure where they get included.

Reply:
Addressed by removing configuration file for Doxygen (./Doxyfile) from spell check.